--- origsrc/sawfish-pager-0.90.0/Makefile.in	2010-10-24 14:45:49.000000000 -0500
+++ src/sawfish-pager-0.90.0/Makefile.in	2011-03-07 23:56:57.060790600 -0600
@@ -23,6 +23,9 @@
 # along with sawfish-pager; see the file COPYING.   If not, write to
 # the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
 
+VPATH = @srcdir@
+srcdir = @srcdir@
+
 CFLAGS = @GTK_CFLAGS@ @IMAGE_LOADER_CFLAGS@ #-DG_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGDK_PIXBUF_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED
 LDFLAGS = -Wl,-rpath @SAWFISH_CLIENT_LIB_DIR@ @SAWFISH_CLIENT_LIB_DIR@/client.so @GTK_LIBS@ @SAWFISH_LIBS@ @IMAGE_LOADER_LIBS@
 
@@ -34,8 +37,8 @@
 install: all
 	mkdir -p  $(DESTDIR)@PAGER_C_DIR@
 	mkdir -p $(DESTDIR)@PAGER_JL_DIR@
-	install -m644 pager.jl $(DESTDIR)@PAGER_JL_DIR@
-	install -m644 pager.jlc $(DESTDIR)@PAGER_JL_DIR@
+	install -m644 $(srcdir)/pager.jl $(DESTDIR)@PAGER_JL_DIR@
+	install -m644 $(srcdir)/pager.jlc $(DESTDIR)@PAGER_JL_DIR@
 	install -m755 sawfishpager $(DESTDIR)@PAGER_C_DIR@
 
 uninstall:
